luda-editor/cdk/lib/cdk-stack.ts

```diff
@@ -110,6 +110,8 @@ export class CdkStack extends cdk.Stack {
                     ],
                 }),
             ],
+            timeout: cdk.Duration.seconds(30),
+            memorySize: 512,
```

Comment:
How much time taken for this?
Increasing memory will get more cpu too so it would be faster and effective, economic.
Benchmark is the best way to optimize, but optimization is not a essential right now.
I just want the information how much it takes with current setting.

Reply:
> It seems to be problem that occurs because `complete_psd` is too heavy a task. In lambda log, runtime exited because of timeout. after increase timeout and memory size, It worked. for [오하연-놀람혹은히익-사복(test).psd](https://drive.google.com/file/d/112GoZO9L25aKv8OPJ_DU10_BvKDO1ymN/view?usp=drive_link), It tooks 6684ms and 266MB.

It tooks 6684ms and 266MB for [오하연-놀람혹은히익-사복(test).psd](https://drive.google.com/file/d/112GoZO9L25aKv8OPJ_DU10_BvKDO1ymN/view?usp=drive_link)

Reply:
no test with 512?

Reply:
That is result of 30sec and 512M limits.

Reply:
You mean it made a timeout with 512m memory limit?

Reply:
FYI
https://docs.aws.amazon.com/lambda/latest/operatorguide/computing-power.html
![image](https://github.com/NamseEnt/namseent/assets/3580430/0ba74aa4-8a1b-4c51-a798-13b3b8c919ee)

Reply:
No, I set time limit to 30sec, and set memory limit to 512m.
Then it worked and result was
> It tooks 6684ms and 266MB for [오하연-놀람혹은히익-사복(test).psd](https://drive.google.com/file/d/112GoZO9L25aKv8OPJ_DU10_BvKDO1ymN/view?usp=drive_link)

Reply:
with 512MB, it's 6sec, right?
Test with 4 cores or 6 cores. it would be cheaper than 1 core with 512MB.

Reply:
I never used this tool but I know it's very super powerful to find best memory settting for lambda  https://github.com/alexcasalboni/aws-lambda-power-tuning

Reply:
[Result of 512~3008](https://lambda-power-tuning.show/#AAIVAyoEQAVVBmoHqgrACw==;VcG1RatydkWrgjFFq6IURQCg60Sr+tJEALDLRFWVwkQ=;CPJMOP0hVjhlZlA4OQVcOGxDUjjPdlw45BqZOA8koTg=)

I can't test 3008+ due to quota. 
> New AWS accounts have reduced concurrency and memory quotas. AWS raises these quotas automatically based on your usage.

https://docs.aws.amazon.com/lambda/latest/dg/gettingstarted-limits.html

Reply:
Additionally, I tested a light api, `list_project_sequences`.
In this case, lower memory is better. [result](https://lambda-power-tuning.show/#AAIVAyoEQAVVBmoHqgrACw==;q6qWQauqhkEAAJBBVVU1QauqWkFVVYVBq6pSQauqUkE=;hF4rNNFIbDQ2Aqk0WQ6ONL7jxzSfGQ41VVIoNUl2OTU=)

However, compared to `complete_psd`, the impact does not seem significant. [compare](https://lambda-power-tuning.show/#AAIVAyoEQAVVBmoHqgrACw==;VcG1RatydkWrgjFFq6IURQCg60Sr+tJEALDLRFWVwkQ=;CPJMOP0hVjhlZlA4OQVcOGxDUjjPdlw45BqZOA8koTg=;AAIVAyoEQAVVBmoHqgrACw==;q6qWQauqhkEAAJBBVVU1QauqWkFVVYVBq6pSQauqUkE=;hF4rNNFIbDQ2Aqk0WQ6ONL7jxzSfGQ41VVIoNUl2OTU=;complete_psd;list_project_sequences
).


In conclusion, I think it is good to set the memory size to 1621.

Reply:
They said there are cpu ceiling for lambda with memory limit https://stackoverflow.com/a/66523153
hmm. OK. I confirm 1621 memory size